bundles/binding/org.openhab.binding.snmp/README.md

```diff
@@ -91,3 +91,11 @@ transform/SwitchState.map
 1=OFF
 -=undefined
 ```
+
+This configration displays the port description of port 1 a TP-Link SG-3424 Switch:
```

Comment:
Please drop this sentence and place your new String example beneath the first two examples for Switch and Number.